src/coreclr/vm/ceemain.cpp

```diff
@@ -1326,6 +1326,8 @@ void STDMETHODCALLTYPE EEShutDownHelper(BOOL fIsDllUnloading)
 
         {
             CONTRACT_VIOLATION(ModeViolation);
+            // At the moment, this doesn't do anything more than log statistics.
+            VirtualCallStubManager::UninitStatic();
```

Comment:
You're sure you want to move this out of the `if` below? That ShutDown_Phase2 flag used to be guarding from doing the shutdown work multiple times.

Reply:
The point of moving out is not relying on fIsDllUnloading, but yes to make sure it's executed only once I can add extra condition checking ShutDown_Phase2 flag.

Reply:
Now call is moved to first part of shutdown which I believe is executed only once.